### PR TITLE
fix conversion factor label mg/g

### PR DESCRIPTION
**Pull request by danieleongari**
Fixing https://github.com/yakutovicha/aiida-raspa/issues/36

### CODE REVIEW

**Comment by yakutovicha** [comment]
No idea what happens when you do a PR to this repository: the tests simply do not want to run :)

**Comment by yakutovicha** [comment]
Hm, the test did run, actually.
There is a bug, please fis it: https://travis-ci.org/yakutovicha/aiida-raspa

File: aiida_raspa/utils/base_parser.py [hunk 1]
```diff
@@ -141,9 +141,10 @@ def parse_base_output(output_abs_path, system_name, ncomponents):
             if "Conversion factor molecules/unit cell -> mol/kg:" in line:
                 res_cmp['conversion_factor_molec_uc_to_mol_kg'] = float(line.split()[6])
                 res_cmp['conversion_factor_molec_uc_to_mol_kg_unit'] = "(mol/kg)/(molec/uc)"
-            if "Conversion factor molecules/unit cell -> gr/gr:" in line:
-                res_cmp['conversion_factor_molec_uc_to_gr_gr'] = float(line.split()[6])
-                res_cmp['conversion_factor_molec_uc_to_gr_gr_unit'] = "(gr/gr)/(molec/uc)"
+            if "Conversion factor molecules/unit cell -> gr/gr:" in line \ # typo corrected in commit c1ad4de (Nov19)
```
Comment (yakutovicha) [inline aiida_raspa/utils/base_parser.py]: Even GitHub complains about it

**Comment by danieleongari** [comment]
See https://github.com/yakutovicha/aiida-raspa/pull/38

**Pull request closed without merging**